[Feat] 가장 최근에 실패한 5개의 챌린지 조회 api 구현

Jinho622 opened the pull request:
## #️⃣연관된 이슈
> #110 

## 📝작업 내용
> 가장 최근에 실패한 5개의 챌린지 조회 api 구현

## 🔎코드 설명
> 가장 최근에 실패한 5개의 챌린지 조회 api 구현
> ![image](https://github.com/user-attachments/assets/4e3e3cd6-fb6d-49f3-8b18-f6ac3cffd4a1)

## 💬고민사항 및 리뷰 요구사항 (Optional)
> x

## 비고 (Optional)
> x

Jinho622 committed:
feat: #110 가장 최근에 실패한 5개의 챌린지 조회 api 구현 완료

yoonsu417 commented on src/main/java/TtattaBackend/ttatta/web/dto/ChallengeResponeseDTO.java:
```diff
@@ -20,4 +20,23 @@ public static class CreateChallengeResultDTO {
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class FailChallengeListResultDTO {
+        List<ChallengeResponeseDTO.FailChallengeResultDTO> failChallengeList;
+    }
```
여기는 List FailChallengeResultDTO 로 가도 괜찮을 것 같아요

Jinho622 replied:
ㅋㅋㅋㅋㅋ오늘까지 api를 끝내야돼서..
아 넵 수정해놓겠습니다

yoonsu417 reviewed (approved):
놀아...노트북을 닫아...

Jinho622 committed:
refactor: #110 실패한 챌린지 조회할때 금일 챌린지는 제외 시키는 기능 추가 구현 및 ChallengeResponseDTO수정

ChallengeResponseDTO수정사항
- FailChallengeListResultDTO의 failCahllengeList의 List타입 수정

Jinho622 committed:
Merge branch 'develop' into feature/#110-get-fail-challenge
